### DISCUSSION TITLE
Can't close Cider

### DISCUSSION THREAD

**Discussion by gitchat1** [ciderapp/Cider#539] [Q&A] [answered]
Hi!

The app looks extremely promising so far. I just can't close it unless I kill it via the task manager. It would be nice if I could quit the app without having to resort to task manager.

**Comment by tva2000hn** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/27716185/156369858-523d7f33-e427-4697-99c4-1d0b83e7f843.png)
turn this off

**Comment by gitchat1** [comment]
Thanks Any Idea how I can comvert my Spotify-Playlists reliably?
